images/openshift-ci/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This Dockerfile is used by openshift CI
+# It builds an image containing snc code and nss-wrapper for remote deployments, as well as the google cloud-sdk for nested GCE environments.
+FROM scratch AS builder
+WORKDIR /code-ready/snc
+COPY . .
+
+FROM centos:7
```

Comment:
centos:8? I'm only running snc.sh on rhel8/fedoras, so I don't know how well el7 works.

Reply:
centos 7 works... so should centos 8, but anyhing particular reason?

Reply:
If Praveen is not testing el7 on a regular basis, this means it's almost never tested on el7, so we could hit unexpected issues. In a way that's good, but given rhel7 age, I'm not sure it's really worth making sure el7 works in addition to el8.

Reply:
This image just carry the snc code not going to run it on el7 but run it on GCP VM which is rhel8 so I don't care what we have here, if you folks want `8` then I can change it.

Reply:
I'd explain in the commit log how this image is going to be used then.